### DISCUSSION TITLE
Help with Flatpak install on 7.0

### DISCUSSION THREAD

**Discussion by KerryatCLC** [elementary/triage#276] [Q&A]
I went to Flathub and clicked install on LibreOffice. Nothing showed up or indicated it had installed. I then attempted to run the two commands shown at the INSTALL dropdown. Still no LibreOffice.

**Comment by KerryatCLC** [comment]
Instructions say to use Sideload to to open d/l'd .flatpakref file. I don't see the Sideload app anywhere.

**Comment by KerryatCLC** [comment]
I found it in the app store with options to uninstall or open. I clicked open and nothing happened.

**Reply by davidmhewitt** [reply]
What happens if you click on the downloaded flatpakref file in either the file manager or the downloads list of your browser?
